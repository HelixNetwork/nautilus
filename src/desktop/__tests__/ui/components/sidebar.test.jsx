import React from 'react';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(<li page="settings/language">language</li>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
it('renders correctly', () => {
    const tree = renderer
        .create(<li page="settings/theme">Theme</li>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
it('renders correctly', () => {
    const tree = renderer
        .create(<li page="settings/accountsetting">Advanced settings</li>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
it('renders correctly', () => {
    const tree = renderer
        .create(<li page="settings/editname">Edit Name</li>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
it('renders correctly', () => {
    const tree = renderer
        .create(<li page="settings/node">Node</li>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
it('renders correctly', () => {
    const tree = renderer
        .create(<li page="settings/viewseed">ViewSeed</li>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
it('renders correctly', () => {
    const tree = renderer
        .create(<li page="settings/address">View Address</li>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
it('renders correctly', () => {
    const tree = renderer
        .create(<li page="settings/password">Change password</li>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
it('renders correctly', () => {
    const tree = renderer
        .create(<li page="settings/remove">Remove Account</li>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
it('renders correctly', () => {
    const tree = renderer
        .create(<li page="settings/snapshot">Snapshot</li>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});