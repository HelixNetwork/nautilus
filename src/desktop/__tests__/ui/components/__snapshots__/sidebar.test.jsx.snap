// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  page="settings/language"
>
  language
</li>
`;

exports[`renders correctly 2`] = `
<li
  page="settings/theme"
>
  Theme
</li>
`;

exports[`renders correctly 3`] = `
<li
  page="settings/accountsetting"
>
  Advanced settings
</li>
`;

exports[`renders correctly 4`] = `
<li
  page="settings/editname"
>
  Edit Name
</li>
`;

exports[`renders correctly 5`] = `
<li
  page="settings/node"
>
  Node
</li>
`;

exports[`renders correctly 6`] = `
<li
  page="settings/viewseed"
>
  ViewSeed
</li>
`;

exports[`renders correctly 7`] = `
<li
  page="settings/address"
>
  View Address
</li>
`;

exports[`renders correctly 8`] = `
<li
  page="settings/password"
>
  Change password
</li>
`;

exports[`renders correctly 9`] = `
<li
  page="settings/remove"
>
  Remove Account
</li>
`;

exports[`renders correctly 10`] = `
<li
  page="settings/snapshot"
>
  Snapshot
</li>
`;
